cli/certinfo/certinfo.go

```diff
@@ -21,12 +24,14 @@ Usage of certinfo:
         cfssl certinfo -csr file
 	- Data from certificate from remote server.
         cfssl certinfo -domain domain_name
+    - Data from CA storage
```

Comment:
Whitespace looks slightly off here

Reply:
Oh, such a rookie mistake! 🤦‍♂️ Didn't see the mix of tabs and spaces. Will fix!